docker-run-generate-outputs.sh

```diff
@@ -86,26 +89,40 @@ fi
 docker start -a -i "$container"
 
 # Copy the output data back out of the container
+echo "copying icr files to "$OUTPUT_ICR_DIR" "
```

Comment:
Nice :)

Reply:
Although it looks like you're trying to put "" inside other "", which won't work (in which case use '' or \"\")

Reply:
So GitHub destroyed all my helpful formatting here. But I hope you get the point.